es/synonyms/ru.txt

```diff
@@ -0,0 +1,156 @@
+# authors
```

Comment:
Nice, How did you check this works well?

Reply:
we checked with Menuchin